test/integration/graphql_test.lua

```diff
@@ -192,3 +192,67 @@ function g.test_error_extensions()
         'Invalid cluster topology config'
     )
 end
+
+function g.test_middleware()
+    local server = cluster.main_server
+
+    -- GraphQL execution can be interrupted by raising error in trigger
+    server.net_box:eval([[
+        local errors = require('errors')
+        graphql = require('cartridge.graphql')
+        function raise()
+            local err = errors.new('E', 'You are not welcome here')
+            err.graphql_extensions = {code = 403}
```

Comment:
This feature is described nowhere

Reply:
There is no place to describe it yet, it's a debt. I've mentioned it in #542